_posts/2023-12-07-semantic-options-benchmarks.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Semantic Search with OpenSearch: Architecture options and Benchmarks"
+authors:
+ - sean-zheng-amazon
+ - zane-neo
+ - ylwu-amzn
```

Comment:
'ylwu-amzn' author profile does not exist - should it be 'ylwu'? Please confirm/fix